coralproject/talk
# Issue-676 add addOnBlur preventing loss of domain-whitelist settings

## What does this PR do?
- set props `addOnBlur` true from the react-tagsinput 
- typed in domain-whitelist settings e.g. `localhost:3000 `will be set in Configure->Tech Settings->Permitted Domains 
- this address #676

## How do I test this PR?
- reset or maybe better remove Talk MongoDB database e.g.
```
> use talkdb
> db.dropDatabase()
```
- open http://localhost:3000/admin/install

- fill forms until point "3. Domain Whitelist"

- insert "localhost:3000" by typing, do not copy/paste nor hit the return Key!

- click button "Finish Install"

- open http://localhost:3000/admin/configure

- click "Tech Settings" and compare Permitted Domains

- verify the form shows `localhost:3000`
